### PR TITLE
[WIPTEST] CRUD tests for Networks

### PR DESCRIPTION
**Pull request by quarckster**
Purpose
=================

CRUD tests for Networks tab. Obsoletes #5350.

{{pytest: -v cfme/tests/networks/test_components_crud.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit da6ac3f026eb6d3f141321a0f307f73d44013792:

`cfme/tests/networks/test_components_crud.py`:
- :red_circle: Line 8:1: **F401** *'cfme.utils.wait.wait_for' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 36d9bea6a61e644b9d8a1792733bed92e340eed4:

`cfme/networks/network_router.py`:
- :red_circle: Line 11:1: **F401** *'cfme.utils.blockers.BZ' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5c2c881ee809a0cf93670e2412abd1aed8b8fdb6:

`cfme/networks/security_group.py`:
- :red_circle: Line 82:19: **F821** *undefined name 'SecurityGroupsNotFound'*
- :red_circle: Line 88:12: **F821** *undefined name 'SecurityGroupAddView'*
- :red_circle: Line 89:20: **F821** *undefined name 'NavigateToSibling'*
- :red_circle: Line 95:13: **F821** *undefined name 'BZ'*

`cfme/tests/networks/test_components_crud.py`:
- :heavy_exclamation_mark: Line 154:1: **W391** *blank line at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud/test_security_groups.py [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
-        * Provide Security groups Description.
-        * Select Cloud Tenant.
-        * Cancel it.
-    """
-    security_group = appliance.collections.security_groups
-    sec_group = security_group.create(name=fauxfactory.gen_alphanumeric(),
-                                      description=fauxfactory.gen_alphanumeric(),
-                                      provider=provider,
-                                      cancel=True)
-    assert not sec_group.exists
```
Comment (quarckster) [inline cfme/tests/cloud/test_security_groups.py]: That code moved to cfme/tests/networks/test_components_crud.py

File: cfme/cloud/security_groups.py [hunk 2]
```diff
@@ -1,247 +0,0 @@
...
-    prerequisite = NavigateToSibling("All")
-
-    def step(self, *args, **kwargs):
-        """Raises DropdownItemDisabled from widgetastic_patternfly
-        if no RHOS Network manager present"""
-        # Todo remove when fixed 1520669
-        if (BZ(1520669, forced_streams='5.9').blocks and
-                self.prerequisite_view.flash.messages):
-            self.prerequisite_view.flash.dismiss()
-        self.prerequisite_view.toolbar.configuration.item_select('Add a new Security Group')
```
Comment (quarckster) [inline cfme/cloud/security_groups.py]: That code moved to cfme/networks/security_group.py

**Comment by dajoRH** [comment]
Lint report for commit 37e5f9bdb99bc0c34e0a97372d6d0379869c5121:

`cfme/networks/security_group.py`:
- :red_circle: Line 36:37: **F821** *undefined name 'SecurityGroupAllView'*
- :red_circle: Line 41:13: **F821** *undefined name 'wait_for'*
- :red_circle: Line 55:16: **F821** *undefined name 'ItemNotFound'*
- :red_circle: Line 91:33: **F821** *undefined name 'SecurityGroupAllView'*
- :red_circle: Line 97:13: **F821** *undefined name 'wait_for'*

`cfme/tests/networks/test_components_crud.py`:
- :red_circle: Line 9:1: **F401** *'cfme.utils.wait.wait_for' imported but unused*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**